src/main/java/org/sonar/plugins/checkstyle/metadata/CheckstyleMetadata.java

```diff
@@ -254,15 +323,23 @@ private static String convertName(String name) {
         return result.toString();
     }
 
+    /**
+     * Create internal key composed of parents of module.
+     *
+     * @param moduleDetails module metadata
+     * @return internalKey
+     */
     private static String convertInternalKey(ModuleDetails moduleDetails) {
         String result = "Checker/";
-        if ("Checker".equals(moduleDetails.getParent())) {
+        if (moduleDetails.getParent().contains("Checker")) {
```

Comment:
why `contains`?

can we keep `equals` ?

Reply:
Done